CHANGELOG.md

```diff
@@ -16,6 +16,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Changed
 
 - [C,D] Updated Substrate to polkadot-v1.1.0
+- [C,D] Introduction of the OpenGov
```

Comment:
We usually put the changes under [vNext] and on release creation tag it with a concrete release

Reply:
We haven't released the `53000` yet that was already merged to the `dev` in this [PR](https://github.com/Cerebellum-Network/blockchain-node/pull/281). That's why I didn't put the change to vNext in order to not skip the `53000` accidentally.